examples/http-server/docker-compose.yaml

```diff
@@ -0,0 +1,69 @@
+version: '3.8'
+
+services:
+  gofr-http-server:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    environment:
+      - TRACE_EXPORTER=gofr
+      - TRACER_RATIO=0.1
+      - REDIS_HOST=redisdb
+      - REDIS_PORT=6379
+      - DB_HOST=mysqldb
+      - DB_USER=root
+      - DB_PASSWORD=password
+      - DB_NAME=test
+      - DB_PORT=3306
+      - DB_DIALECT=mysql
+    ports:
+      - "9000:9000"
+      - "2121:2121"
+    depends_on:
+      - redisdb
+      - mysqldb
+      - grafana
+      - prometheus
```

Comment:
were you able to see the metrics in the prometheus or grafana dashboard?
as it is not appearing for me

Reply:
its visible in grafana dashboard
<img width="1728" alt="Screenshot 2024-10-09 at 8 40 23 PM" src="https://github.com/user-attachments/assets/3a92066a-f034-4d11-969f-539454ef3a17">
<img width="1728" alt="Screenshot 2024-10-09 at 8 40 57 PM" src="https://github.com/user-attachments/assets/ee687883-abee-40f3-8813-d7d6a8d89855">
<img width="1728" alt="Screenshot 2024-10-09 at 8 40 57 PM" src="https://github.com/user-attachments/assets/90f8b110-7718-4de2-b893-2c08a3fce670">

Reply:
my bad, I was trying to test in GitHub code space, probably it had some port forwarding issue.
It works fine in local